### ISSUE TITLE
Flutter web: Cannot not pass data to API(Server) with method POST

### ISSUE THREAD

**Issue by vannam22081988** [flutterchina/dio#1024]
### New Issue Checklist

With dio: ^3.0.10: 
- I use post method to pass data from web to server but body is empty.
- I use http lib with post method to pass data from web to server  then body is not empty.
Please fix this issue on web. 

### Issue Info

| Info                           | Value                                                 |      |
| ------------------------------ | ----------------------------------------------------- | ---- |
| Platform Name                  | e.g. flutter / ios / android                          |      |
| Platform Version               | e.g.  1.5.0 / 12.0 / 9.0                              |      |
| Dio Version                    | e.g.  2.1.0 / 1.0.17                                  |      |
| Android Studio / Xcode Version | e.g. Android Studio 3.3.2 / Xcode 10.2.1              |      |
| Repro rate                     | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo prj        | e.g. does it happen with our demo project?            |      |
| Demo project link              | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by Hannnes1** [comment]
It seems like this issue only appears one some browsers. I have noticed it on Firefox, but not Chrome
